src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCode.cs

```diff
@@ -223,8 +223,7 @@ public static int OpcodeSize(int opcode)
             }
         }
 
-#if DEBUG
```

Comment:
why remove #if DEBUG?

Reply:
Because the code is now used as part of the generator in release.  I could change the condition to be something like `#if DEBUG || REGEXGENERATOR`, but the linker will trim it out anyway if nothing uses it, so it didn't seem worth it.